great_expectations/validator/v1_validator.py

```diff
@@ -83,6 +83,11 @@ def validate_expectation_suite(
             },
         )
 
+    @property
+    def active_batch_id(self) -> Optional[str]:
+        # See validation_operators line 343
```

Comment:
This feels like it might be outdated pretty easily (especially since I'm hoping to remove validation operators). Is there a less specific comment we can use?